tests/benchmarks/i2c_endless/testcase.yaml

```diff
@@ -8,9 +8,11 @@ common:
   platform_allow:
     - nrf52840dk/nrf52840
     - nrf5340dk/nrf5340/cpuapp
+    - nrf54l15dk/nrf54l15/cpuapp
```

Comment:
Support for PDK can be remove for this tests - no need to keep pdk files, only dk can stay.

Reply:
done